config/secrets.yml

```diff
@@ -10,16 +10,91 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
+default: &default
+  subject_prefix_for_outgoing_emails: WHEEL
+  mailer_default_from_email: "'Wheel' <notifications@wheel.com>"
+  mailer_delivery_method: :smtp
+
+  twilio:
+    from_number: <%= ENV['TWILIO_FROM_NUMBER'] %>
+    account_sid: <%= ENV['TWILIO_ACCOUNT_SID'] %>
+    auth_token: <%= ENV['TWILIO_AUTH_TOKEN'] %>
+
+  aws_s3:
+    access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+    secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+    bucket_name: <%= ENV['AWS_S3_BUCKET_NAME'] %>
+
+  support_email: support@wheel.com
+  host: <%= ENV['APP_URL'] || ENV['HEROKU_URL'] || 'http://localhost:3000' %>
+
 development:
+  <<: *default
   secret_key_base: 3193ce62de10f78da223e95760dd6c08523cd78204b4d8dc8bb119faf8847adceb70aadce1288ffd8e6232afb3a704c60a75feb024ab937de989322f4e64a385
+  intercept_and_forward_emails_to:
+    - john@example.com
+    - adam@example.com
+  devise:
+    secret_key: fc65e594fdad453630db5238ea7398dee52e788feaf71d1c99d4bb8b443b4d297505473d01ad7a09ccbf02be2da9b9f12ea89998bfe3fee9a24cc6dedb3ea350
+  mailer:
+    smtp_settings:
+      user_name: <%= ENV['MANDRILLAPP_USERNAME'] %>
+      password: <%= ENV['MANDRILLAPP_PASSWORD'] %>
+      address: smtp.mandrillapp.com
+      port: 587
+      authentication: plain
+  devise:
+    secret_key: c0cd8c380935936ff6c6bfb7b0f0ed3f3a2d66d7d7b445394a879dea71bfb5711799c7840e7d4fd93fe8a0983c4f27fd30caebea879dff734c15ff1696262144
 
 test:
-  secret_key_base: b731f261415e6f13e3e3e5a6726ee163707938f38514331fc2a67eda3800dd428bfe2a0c3b4ed83cf99e7df643e93b450511f81e05556712128ae786559ada6e
+  <<: *default
+  secret_key_base: 08523cd7820d8e6232afb3a704c60a75feb024ab937de989322f4e64a3854b4d8dc8bb119faf8847adceb70aadce1288ff3193ce62de10f78da223e95760dd6c
+  mailer_delivery_method: :test
+  mailer:
```

Comment:
It can differ, compare across this file. Some use auth, some use starttls, etc. Some don't.
For testing as well as development, I find this handy, because we don't need to copy over the settings to test out delivery.